CONTRIBUTING.md

```diff
@@ -252,6 +256,18 @@ Python world. If you make it through even some of these guides, you will be well
 on your way to being a “Pythonista” (a Python developer) writing “Pythonic”
 (canonically correct Python) code left and right.
 
+### Async IO
+
+With Python 3.4, the Async IO pattern found in languages such as C# and Go is
```

Comment:
Go doesn't really have an async I/O pattern; everything is written as if it were blocking code.  The asyncness of it happens under the hood, inside its runtime libraries.